### PR TITLE
Feature/windows builds

### PR DESCRIPTION
**Pull request by UnseenWizzard**
Makes it possible to run all Make targets on windows (gnuwin make). 

Also updates CONTRIBUTING.md to only reference the make targets rather than documenting possibly outdated go commands

### CODE REVIEW

**Review by tatomir146 — APPROVED** [review]

**Review by didiladi — APPROVED** [review]

**Comment by UnseenWizzard** [comment]
@patrickpichler I'd appreciate a look from you as well! after all you're the biggest Makefile proponent and so far sole contributor to it ;)

File: Makefile [hunk 1]
```diff
@@ -1,24 +1,41 @@
...
+	@.\tools\check-format.cmd
+else
 	@sh ./tools/check-format.sh
+endif
 
 format:
 	@gofmt -w .
 
-build: clean
+mocks:
```
Comment (patrickpichler) [inline Makefile:15]: You need to add `mock` also to the `.PHONY` tasks

**Review by patrickpichler — COMMENTED** [review]
As I do not have a windows machine, I cannot really test it. Would be nice if somebody with a windows machine could test it.

File: Makefile (hunk 1 above)
Comment (UnseenWizzard) [reply Makefile:15]: thx, missed that one

**Comment by UnseenWizzard** [comment]
> As I do not have a windows machine, I cannot really test it. Would be nice if somebody with a windows machine could test it.

I developed this on my windows PC while I was waiting for something to download ;) 

Tested on Win 10 & WSL Ubuntu 20

File: Makefile (hunk 1 above)
Comment (UnseenWizzard) [reply Makefile:15]: Fixed

**Review by patrickpichler — APPROVED** [review]
Would be nice if you could squash the commits ^^

**Pull request merged**